snooty/parser.py

```diff
@@ -446,7 +447,7 @@ def handle_directive(
             # Calculate the expected number of columns for this list-table structure.
             expected_num_columns = 0
             if "widths" in options:
-                expected_num_columns = len(options["widths"].split(" "))
+                expected_num_columns = len(re.split("[,* *]+", options["widths"]))
```

Comment:
I believe that this regex would permit multiple commas in a row, which we may want to avoid? In my opinion `:widths: 38,,72` should not be treated the same as `:widths: 38,72`. What do you think?

Additionally [`\s`](https://docs.python.org/3/library/re.html#index-30) may better represent whitespace, as it is a little more forgiving than `' '`

Reply:
Agreed! Updating